src/shared/components/hero-section/index.js

```diff
@@ -1,26 +1,166 @@
-import { FormattedMessage } from 'react-intl'
+import React, { Component } from 'react'
+import { FormattedMessage, injectIntl } from 'react-intl'
+import { PropTypes } from 'prop-types'
+import axios from 'axios'
+import classNames from 'classnames'
+import { distanceInWordsToNow } from 'date-fns'
+import locales from 'utils/dateFnsLocales'
 
+import OrbitsSvg from 'shared/media/backgrounds/orbits.svg'
 import Button from 'shared/components/button'
 import cubePng from 'shared/media/images/cube.png'
 import styles from './index.module.css'
 
-const Hero = () => (
-  <div className={styles.container}>
-    <div className={styles.content}>
-      <img src={cubePng} />
-      <FormattedMessage tagName='h1' id='heroWelcomeMessage' />
-      <FormattedMessage tagName='p' id='heroTextDescription' />
-      <div className={styles.infoContainer}>
-        <FormattedMessage tagName='span' id='heroCurrentVersion' />
-        <FormattedMessage tagName='span' id='heroLatestUpdate' />
-        <FormattedMessage tagName='span' id='heroDownloadsLastMonth' />
-      </div>
-      <div className={styles.buttonContent}>
-        <Button translationId='buttonLearnMore' path='test' />
+class Hero extends Component {
+  constructor () {
+    super()
+
+    this.state = {
+      info: undefined,
+      errorMessage: undefined
+    }
+  }
+
+  componentDidMount () {
+    const self = this
+    this._ismounted = true
+
+    axios.get('https://api.npms.io/v2/package/ipfs')
+      .then(function ({ data: { collected } }) {
+        self.handleAxiosResponse(collected)
+      })
+      .catch(function (error) {
+        self.handleAxiosError(error)
+      })
+  }
+
+  componentWillUnmount () {
+    this._ismounted = false
+  }
+
+  render () {
+    const { info, errorMessage } = this.state
+    const isDataLoaded = Boolean(info)
+    const existsError = Boolean(errorMessage)
+    const infoContainerClasses = classNames(styles.infoContainer, {
+      [styles.hidden]: !isDataLoaded && !existsError,
+      [styles.error]: !isDataLoaded && existsError
+    })
+
+    return (
+      <div className={ styles.container }>
+        <div className={ styles.orbitContainer } >
+          <div className={ styles.orbits }> <OrbitsSvg /> </div>
+        </div>
+        <div className={ styles.content }>
+          <img src={ cubePng } />
+          <FormattedMessage tagName="h1" id="heroWelcomeMessage" />
+          <FormattedMessage tagName="p" id="heroTextDescription" />
+          <div className={ infoContainerClasses }>
+            { isDataLoaded && !existsError ? this.renderPkgInfo(info, isDataLoaded) : this.renderErrorMessage(errorMessage) }
+          </div>
+          <div className={ styles.buttonContent }>
+            <Button translationId="buttonLearnMore" path="test" />
+          </div>
+        </div>
       </div>
+    )
+  }
+
+  renderPkgInfo = (info, isDataLoaded) => (
+    <div>
+      <span>{ isDataLoaded ? info.currentVersionStr : '' }</span>
```

Comment:
You should just compare once because the result is the same when no data was loaded.